2 data pet grooming biz scenario.sql

```diff
@@ -0,0 +1,28 @@
+Use PetgroomingDB
+GO
+
+DELETE Petgroom
+go 
+
+insert Petgroom(PetOwnerFirstName, PetOwnerLastName, PetOwnerAddress, PetType, PetName, PricePerGrooming, groomingfrequency, DatePickedUp, PreviousCustomer )
```

Comment:
Nice job! All dates should be changed to a proper date format ie: d/m/y or y-m-d...